### PR TITLE
cleanup: remove unused Markets model

### PR DESCRIPTION
**Pull request by robertkowalski**
data can be stored in a simple `Map`, model was removed in
hf-server

https://github.com/bitfinexcom/bfx-hf-server/pull/97

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,7 +28,6 @@ Besides these, two DB backends are available:
 * Backtest
 * Candle
 * Credential
-* Market
```
Comment (avsek477) [inline README.md:31]: one remain in line 80.. it isn't used anyway

Comment (robertkowalski) [reply README.md:31]: good catch! fixed!

**Review by avsek477 — APPROVED** [review]
lgtm

**Review by ghost — APPROVED** [review]
lgtm

**Review by tarcisiozf — APPROVED** [review]
lgtm

**Pull request merged**